lib/appmgr/app-operator-pid

```diff
@@ -177,10 +177,15 @@ command_stop() {
       ;;
   esac
 
-  echo -n "Sending TERM to $PID, waiting for shutdown"
+  local stop_timeout=$(app-conf get app.stop_timeout)
```

Comment:
this does not seem to work, so I'll test some more and get back with an updated PR